packages/flutter/test/material/scaffold_test.dart

```diff
@@ -70,17 +71,17 @@ void main() {
 
     scaffoldState.openDrawer();
     await tester.pumpAndSettle();
-    expect(true, isDrawerOpen);
+    expect(isDrawerOpen, true);
```

Comment:
Thanks for cleaning these up!

Reply:
Sure, updated docs.

I took this solution from https://github.com/flutter/flutter/pull/59310 and it didn't document this behavior as a result I've filed https://github.com/flutter/flutter/issues/106650 to document the same dismission action behavior for `PopupRoute` /`ModalRoute` and some official examples and code snippets. As these widgets proper documentation too.

Reply:
Excellent! Thank you for being so thorough!